bridge_adaptivity/module/engines/engine_vpal.py

```diff
@@ -121,3 +121,14 @@ def sync_collection_activities(self, collection):
         return self.check_engine_response(
             sync_collection.status_code, action='synchronized', obj='collection', name=collection.name, status=201
         )
+
+    def get_grade(self, sequence):
+        url = urlparse.urljoin(self.base_url, 'collection/{collection_id}/grade'.format(
+            collection_id=sequence.collection.id)
+        )
+        response = requests.post(url, json={'learner_id': sequence.owner.id}, headers=self.headers)
+        if self.check_engine_response(response.status_code, status=200):
```

Comment:
status=200 is the default parameter and it could be avoided instead I would add `obj` and `action` params to make logs msg become more informative.